pkgs/by-name/ex/exo/patch-used-ports-file.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/exo/helpers.py b/exo/helpers.py
+index f56b21a..796bf86 100644
+--- a/exo/helpers.py
++++ b/exo/helpers.py
+@@ -34,7 +34,7 @@ def get_system_info():
+ 
+ 
+ def find_available_port(host: str = "", min_port: int = 49152, max_port: int = 65535) -> int:
+-  used_ports_file = os.path.join(os.path.dirname(os.path.abspath(__file__)), ".exo_used_ports")
```

Comment:
This patch shouldn't be needed any more. We use temp dir now for `.exo_used_ports`

Reply:
Indeed, this was from before I was able to upstream to the patch.